src/main/java/edu/wpi/grip/core/Source.java

```diff
@@ -45,4 +46,44 @@
      * @see #getProperties()
      */
     public abstract void createFromProperties(EventBus eventBus, Properties properties) throws IOException;
+
+
+    public <T extends Source> T start(EventBus eventBus) throws IOException {
```

Comment:
This makes it possible to allow you to do `new CameraSource(...).start(eventBus);`